### PR TITLE
Refs #26925 - registration token method naming

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Registration token would use some tweaks.

### CODE REVIEW

File: app/services/foreman_register/registration_token.rb [hunk 1]
```diff
@@ -11,20 +14,20 @@ def encode(host, secret)
       private
 
       def prepare_payload(host, secret)
-        exp = iat + (24 * 3600)
-        nbf = iat - 3600
-        jti_raw = [secret, iat].join(':')
+        expire_at = issued_at + EXPIRE_AFTER_ISSUING
+        not_before = issued_at - VALID_BEFORE_ISSUING
+        jti_raw = [secret, issued_at].join(':')
```
Comment (ofedoren) [inline app/services/foreman_register/registration_token.rb]: If you decided to "unpack" those shortcuts, why you haven't unpacked `jti` as well? :)

Comment (ezr-ondrej) [reply app/services/foreman_register/registration_token.rb]: Yeah, good thinking :+1:

**Review by ofedoren — APPROVED** [review]
I don't think that katello test failis are related..

**Comment by timogoebel** [comment]
[test katello]

**Comment by ezr-ondrej** [comment]
:green_apple:

**Review by timogoebel — APPROVED** [review]

**Comment by timogoebel** [comment]
Thanks, @ezr-ondrej.

**Pull request merged**